spokestack/extensions/webrtc/vad.pyx

```diff
@@ -0,0 +1,41 @@
+cimport cvad
+cimport numpy as np
+
+
+np.import_array()
+
+cdef class WebRtcVad:
+    cdef cvad.VadInst*_vad
+    cdef int _sample_rate
+    cdef int _mode
+
+    def __dealloc__(self):
+        cvad.WebRtcVad_Free(self._vad)
+
+    def __init__(self, sample_rate=16000, mode=0):
```

Comment:
this constructor needs to validate its configuration parameters like the android version
https://github.com/spokestack/spokestack-android/blob/master/src/main/java/io/spokestack/spokestack/webrtc/VoiceActivityDetector.java#L80